lib/functions/buildFileList.sh

```diff
@@ -7,6 +7,14 @@
 ################################################################################
 ########################## FUNCTION CALLS BELOW ################################
 ################################################################################
+
+################################################################################
+#### Function BuildFileList ####################################################
+function IssueHintForFullGitHistory() {
+    info "Check that you have the full git history, the checkout is not shallow, etc"
+    info "See https://github.com/github/super-linter#example-connecting-github-action-workflow"
```

Comment:
```suggestion
  info "Check that you have the full git history, the checkout is not shallow, etc"
  info "See https://github.com/github/super-linter#example-connecting-github-action-workflow"
```

We use two whitespaces elsewhere for indentation.